### DISCUSSION TITLE
Keys with accented letters not working with static data

### DISCUSSION THREAD

**Discussion by gfigueroa-hb** [tolgee/tolgee-platform#1636] [Q&A]
Hi everybody!
I'm using the **Svelte** integration of Tolgee and I noticed that when i use a key that contains an accented letter it doesn't work if the translation is provided from **static data**.
There is a way to solve this issue without removing every accented letter from the keys?

**Comment by marcosanson** [comment]
please, any news on that?

**Comment by stepan662** [comment]
Hello, I failed to replicate. Could you share more details how to reproduce this error?
